Source/Fuse.Reactive.JavaScript/ObservableProperty.uno

```diff
@@ -8,7 +8,7 @@ namespace Fuse.Reactive
 	{
 		public LazyObservableProperty(ThreadWorker w, Scripting.Object obj, Property p): base(w, obj, p)
 		{
-			w.Context.ObjectDefineProperty(obj, p.Name.ToString(), Get);	
+			w.Context.ObjectDefineProperty(obj, JSName, Get);	
```

Comment:
This particular case is incompatible because you now have two observables representing the same value (double source of truth)